day00/ex03/ex03.md

```diff
@@ -0,0 +1,77 @@
+# Exercise 03 - Populate
+|                         |                    |
+| -----------------------:| ------------------ |
+|   Turnin directory :    |  ex03              |
+|   Files to turn in :    |  populate.py       |
+|   Forbidden function :  |  None              |
+|   Remarks :             |  n/a               |
+
+## Objective
+
+You must insert :
+* `appstore_games.normalized.csv`
+* `appstore_games_genres.normalized.csv`
+* `appstore_games_languages.normalized.csv` 
+
+data into a postgreSQL table.
+
+## Instructions
+
+You can read the psycopg2_basics documentation (some included functions will help you for this exercise). You can use any PostgreSQL keywords included in the given documentation.
+
+1) You first need to create 3 functions
+- `create_appstore_games`
+- `create_appstore_games_genres`
+- `create_appstore_games_languages`
+
+... to create the following tables :
+
+![tables](../imgs/tables.png){width=450px}
+
+nb : Foreign keys are a reference to an existing column in another table.
+
+2) You will have to create the 3 populate functions
+
+* **populate_appstore_games**
+* **populate_appstore_games_genres**
+* **populate_appstore_games_languages**
```

Comment:
It is not important but the functions are in bold here but not before on lines 23 to 25, the format you do here is then slightly different

Reply:
remark will be fixed in another PR